src/JSONDB.php

```diff
@@ -327,21 +326,21 @@ private function _update()
                     if (! array_diff_key($this->update, $content)) {
                         $this->content[$i] = (object) array_merge($content, $this->update);
                     } else {
-                        throw new Exception('Update method has an off key');
+                        throw new \Exception('Update method has an off key');
```

Comment:
The previous `...\...\Exception` was missing. This is to use the native one :+1: